explorer/src/pages/Mixnodes/index.tsx

```diff
@@ -39,14 +39,24 @@ export const PageMixnodes: React.FC = () => {
     }
   }, [searchTerm, mixnodes?.data])
 
+  const linkStyles = {
+    color: 'inherit',
+    textDecoration: 'none',
+    marginLeft: 16
```

Comment:
Can we generate this margin from the mui theme? e.g. theme.spacing(2)?

Have you looked at mui's Link component? Not sure if that would work. If it does it would allow the styling to be done via sx prop?